rclpy/test/test_qos_event.py

```diff
@@ -48,7 +47,6 @@ def setUp(self):
         self.node = rclpy.create_node('TestQoSEvent',
                                       namespace='/rclpy/test',
                                       context=self.context)
-        self.is_fastrtps = 'rmw_fastrtps' in get_rmw_implementation_identifier()
```

Comment:
There's a flake8 error because we no longer are using the import for `get_rmw_implementation_identifier`.